### PR TITLE
fixed trip bugs

### PR DESCRIPTION
**Pull request by witehound**

### CODE REVIEW

**Comment by andrew-bierman** [comment]
Looks like create trip throws a 401 when trying to create

**Comment by andrew-bierman** [comment]
Nearby Parks not populating

File: client/store/trailsStore_copy.js [hunk 1]
```diff
@@ -19,16 +18,17 @@ export const fetchTrails = createAsyncThunk(
     if (lon) {
       params += `&lon=${lon}`;
     }
-    const radius = 500;
```
Comment (andrew-bierman) [inline client/store/trailsStore_copy.js:22]: Why no radius ?

Comment (witehound) [reply client/store/trailsStore_copy.js:22]: I was getting more trails result without the radius than with the radius do you think it’s okay if Trails result don’t always come in for every location?

**Pull request merged**